ddsrouter_test/compose/test_cases/stress/single_router/compose.yml

```diff
@@ -0,0 +1,197 @@
+# Test description:
+#   This test simulates a busy network:
+#   If we set up a router with two participants, twelve best-effort publishers
+#   sending large messages to three best-effort subscribers on three different topics
```

Comment:
Please indicate the message sizes and their frequencies, since it's quite relevant.